atomic_reactor/plugins/post_tag_and_push.py

```diff
@@ -169,12 +178,6 @@ def run(self):
                 registry_image.registry = registry
                 max_retries = DOCKER_PUSH_MAX_RETRIES
 
-                expect_v2s2 = False
-                for registry in self.registries:
-                    media_types = self.registries[registry].get('expected_media_types', [])
-                    if MEDIA_TYPE_DOCKER_V2_SCHEMA2 in media_types:
-                        expect_v2s2 = True
-
```

Comment:
The change seems correct, but why was it done like this in the first place?
If I'm not mistaken, this code essentially says `expect V2_SCHEMA2 if any of the registries have it`. Shouldn't it be `expect V2_SCHEMA2 if the current registry has it`?

Reply:
This is also the question in my mind.